### PR TITLE
configure line_length in .formatter

### PR DESCRIPTION
**Pull request by ouven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,5 +35,20 @@ Now run
...
+
+The formatter default the a maximum line_length of 98 characters, which can be overwritten with the `line_lenght` option in the `.formatter` file.
+
+Set `heex_formatter_line_length` to only set the line:lenght for the heex formatter.
+
+```elixir
+[
+  plugins: [HeexFormatter],
+  inputs: ["{config,lib,test}/**/*.{ex,exs}"],
+  heex_formatter_line_length: 300
```
Comment (feliperenan) [inline README.md]:
```suggestion
  heex_line_length: 300
```

Maybe let's call it `heex_line_length`? Wdyt?

File: README.md [hunk 2]
```diff
@@ -35,5 +35,20 @@ Now run
 ```elixir
 mix compile
 ```
+### options
+
+#### line_lenght
+
+The formatter default the a maximum line_length of 98 characters, which can be overwritten with the `line_lenght` option in the `.formatter` file.
```
Comment (feliperenan) [inline README.md:42]:
```suggestion
The formatter default the a maximum line_length of 98 characters, which can be overwritten with the `line_length` option in the `.formatter` file.
```

**Review by feliperenan — COMMENTED** [review]
Amazing ✨  

Thanks for your contribution @ouven 💪🏻 

I would just call the option `heex_line_length` instead. Wdyt?

**Comment by ouven** [comment]
I agree, as it is obviously a formatter 👍

File: README.md [hunk 3]
```diff
@@ -35,5 +35,20 @@ Now run
 ```elixir
 mix compile
 ```
+### options
+
+#### line_lenght
```
Comment (feliperenan) [inline README.md:40]:
```suggestion
#### line_length
```

**Comment by feliperenan** [comment]
Thanks ❤️

**Pull request merged**